src/components/panel/ManifestLabel.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC, useEffect, useState } from 'react'
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover.tsx'
+import { Button } from '@/components/ui/button.tsx'
+import { useTranslation } from 'react-i18next'
+import { CircleAlert } from 'lucide-react'
+
+import { useDataStore } from '@/store/DataStore.tsx'
+import { usePanel } from '@/contexts/PanelContext.tsx'
+import { apiRequest } from '@/utils/api.ts'
+
+
+interface ManifestLabelProps {
+  selectedManifest: Manifest | null,
+  updateSelectedManifest: (newManifest: Manifest | null) => void
+  setShowItemModal: (show: boolean) => void,
+}
+
+const ManifestLabel: FC<ManifestLabelProps> = ({ selectedManifest, updateSelectedManifest, setShowItemModal }) => {
+  const { panelState } = usePanel()
+  const { t } = useTranslation()
+  const collection = useDataStore().collections[panelState.collectionId]?.collection
+  const manifest = panelState.manifest
+  const labels = collection?.sequence.map((item) => item.label)
+  const [showModal, setShowModal] = useState(false)
+  const [selectedLabel, setSelectedLabel] = useState('')
+
+  useEffect(() => {
+    function getManifestLabel() {
+      const label = selectedManifest ? selectedManifest.label : panelState?.manifest?.label ?? null
+      setSelectedLabel(label)
+    }
+
+    getManifestLabel()
+  }, [selectedManifest, manifest])
+
+  async function handleManifestClick(label: string) {
+    const manifestId = collection?.sequence.find((manifest) => manifest.label === label).id
+    const manifest = await apiRequest<Manifest>(manifestId)
+    updateSelectedManifest(manifest)
+    setShowModal(false)
+    setShowItemModal(true)
+  }
+
+  const handleOpenChange = (open: boolean) => {
+    setShowModal(open)
+  }
+
+
+
+  return (
+    <>
+      <Popover open={showModal} onOpenChange={handleOpenChange}>
+        <PopoverTrigger asChild>
+          <Button
+            variant={showModal ? 'secondary' : 'ghost'}
+            className="relative font-semibold text-gray-600"
+            onClick={() =>  setShowModal(!showModal)}
+            data-cy="manifest-label">
+            { selectedLabel }
+            { selectedManifest && <CircleAlert className="absolute mb-4 right-1 text-yellow-300" /> }
```

Comment:
this still looks odd, the border is gray, just check online how such tags are styled.